apps/site/pages/en/learn/getting-started/an-introduction-to-the-npm-package-manager.md

```diff
@@ -18,7 +18,7 @@ It started as a way to download and manage dependencies of Node.js packages, but
 
 ## Packages
 
-`npm` manages downloads of dependencies of your project.
+`npm` installs, updates and manages downloads of dependencies of your project. Dependencies are pre-built pieces of code, such as libraries and packages, that your Node.js application needs to work.  
```

Comment:
let's try a manual fix from the UI here - there are still spaces at the end of the sentence.
```suggestion
`npm` installs, updates and manages downloads of dependencies of your project. Dependencies are pre-built pieces of code, such as libraries and packages, that your Node.js application needs to work.
```